### DISCUSSION TITLE
helloSystem is already installed on my Elitebook 2560p (personal computer) as only OS.

### DISCUSSION THREAD

**Discussion by alphamodh0** [helloSystem/hello#259] [Show and tell]
I have installed helloSystem on my computer and I will use it as a single and daily system. I will try to tell my daily experience as an advanced average user of unix-like systems and I will also tell the experience of my father (a common and ordinary user) and so we can improve in what users are looking for. I will try to help in the process of creating apps, I look forward to the arrival of helloSystem 1.0.
![Elitebook 2560p](https://user-images.githubusercontent.com/90866187/142961478-a43a5139-54a2-4d65-83cc-1f0d50959606.png)

**Comment by probonopd** [comment]
Hello @alphamodh0 thanks for letting us know. We are very interested in how it is going for you as an advanced user (have a peek inside the helloSystem-provided utilities with right-click -> show contents), and especially how it is going for your father.

> I will try to help in the process of creating apps

A great place to start out is to peek around the applications in "Developer Preview", which is shorthand for "we'd like to see someone put some love into them". And of course https://github.com/search?q=org%3AhelloSystem+is%3Aissue+is%3Aopen+label%3A%22help+wanted%22, as well as `FIXME` and `TODO` which are sprinkled liberally throughout the code. Please send small pull requests (addressing one FIXME/TODO at a time), as this makes reviewing them much easier. Thanks, and keep it up!
